OKTA-413125 Overall style edit for OIE embedded auth content - part 1 (#2571)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/aspnet/configorder.md

## Order of the configuration options

When multiple configurations options are used simultaneously, the SDK selects the configuration to use based on the following order:

1. SDK Client constructor
2. Environment variables
3. Configuration

For example, values set in the SDK Client constructor override the
environment variables and configuration file settings. Subsequently,
the environment variables take precedence over the configuration file.

> > **Note:** To avoid confusion with which configuration values that are used by the SDK, you should use only one configuration option in your solution.

Reviewer: ```suggestion
> **Note:** To avoid confusion as to which configuration values are used by the SDK, you should use only one configuration option in your solution.
```
